packages/esm-patient-queue-app/src/patient-queue-metrics/clinic-metrics.component.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import MetricsCard from './metrics-card.component';
+
+import styles from './clinic-metrics.scss';
+import { useTranslation } from 'react-i18next';
+import { useMetrics } from './queue-metrics.resource';
+import { Dropdown, DataTableSkeleton } from 'carbon-components-react';
+import MetricsHeader from './metrics-header.component';
+
+const ClinicMetrics: React.FC = () => {
+  const { t } = useTranslation();
+  const { data: metrics, isError, isLoading } = useMetrics();
+
+  const items = [
+    {
+      id: 'option-1',
+      text: 'Triage',
+    },
+    {
+      id: 'option-2',
+      text: 'Adult return',
+    },
+    {
+      id: 'option-3',
+      text: 'Pharamacy',
+    },
+  ];
+  if (isLoading) {
+    return <DataTableSkeleton role="progressbar" />;
+  }
+
+  return (
+    <>
+      <MetricsHeader />
+      <div className={styles.clinicMetricsContainer}>
+        <MetricsCard
+          totalsLabel={t('patients', 'Patients')}
+          totalsValue={metrics ? metrics.scheduled_appointments : 0}
+          headerLabel={t('scheduledAppointment', 'Scheduled appts. today')}
```

Comment:
```suggestion
          label={t('patients', 'Patients')}
          value={metrics ? metrics.scheduled_appointments : 0}
          title={t('scheduledAppointment', 'Scheduled appts. today')}
```

Reply:
Use `scheduledAppointments` instead.

Reply:
Thanks @denniskigen  and @donaldkibet  for you reviews let me address the suggested changes